components/Hero/src/index.scss

```diff
@@ -198,7 +190,8 @@
     );
   }
 
-  .denhaag-hero--home {
+  .denhaag-hero--home,
```

Comment:
Gescheiden houden!

Reply:
What you did below was applying a `-home` token into `-theme` and `-home`.
Everything that is specifically within the modifier of that hero must be within their own selector.
if the token for `-home` changes you don't want the `theme` to be updated. 
Im defintly not a fan of @extended. One of the reasons is pure readability in the code.  The @extended is used at the link-component. If u check the build-css it's defined 3 times. Therefor, each own modifier has their own tokens.